src/components/Tube.vue

```diff
@@ -0,0 +1,71 @@
+<!-- TODO: move inline styles to css -->
```

Comment:
Still todo?

Reply:
Sort of. I have moved the comment to the tube svg. We should move any styles to css rather than having it inline as it is easier to maintain. We already have this as an issue.